src/app.js

```diff
@@ -6,31 +6,35 @@ angular.module('wBooksApp', [
   'bookItem',
   'booksList',
   'bookDetail',
-  'wInput'
-]).
-config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $urlRouterProvider) {
-  let states = [{ 
-    name: 'booksList', 
-    url: '/books', 
+  'wInput',
+  'signUp'
+]).config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $urlRouterProvider) {
+  var states = [{
+    name: 'booksList',
+    url: '/books',
     component: 'booksList',
     resolve: {
       books: (Books) => {
         return Books.get();
       }
     }
-  }, { 
-    name: 'bookDetail', 
-    url: '/books/{id}', 
+  }, {
+    name: 'bookDetail',
+    url: '/books/{id}',
     component: 'bookDetail',
     resolve: {
       book: (Books, $stateParams) => {
         return Books.get().then(data => {
-          return data.find(book => book.id == $stateParams.id);
-        })
+          return data.find(book => book.id === +$stateParams.id);
```

Comment:
why this plus symbol?

Reply:
$stateParams.id is numeric string, with + symbol convert implicit to number to compare with book.id.